root/etc/e-smith/events/actions/restore-data-restic

```diff
@@ -78,16 +78,20 @@ if ($restic_repository eq '') {
 
 push(@opts, '--repo', $restic_repository);
 
-my @cmd = (CMD, @opts, 'restore', $time, '--target', $position);
+my $ret;
 
 if (@files) {  # restore selected files
+    my @cmd = (CMD, @opts, 'restore', $time, '--target', $position);
     foreach (@files) {
         push(@cmd, '--include', $_);
     } 
+    $ret = system(@cmd);
+} else {
+    # use tar to restore files with non-numeric uid/gid
+    my @cmd = (CMD, @opts, 'dump', $time, '/', "| tar -xvf - -C $position");
```

Comment:
The `v` option for tar allows to print the restored files inside the log (or the stdout).
I think it could be useful but it potentiality create a very big log. 

Should we avoid such output? What do you think @filippocarletti @DavidePrincipi ?

Reply:
I guess the log output hits rsyslog rate limit and excessive lines are omitted automatically. We can keep the *verbose* flag :+1: